src/ui/public/timepicker/__tests__/parse_relative_parts.js

```diff
@@ -0,0 +1,105 @@
+import { parseRelativeString, parseRelativeParts } from '../parse_relative_parts';
+import { expect } from 'chai';
```

Comment:
Have we decided to move to Chai? Afaik it's just used in Timelion. I don't wanna be a killjoy but... it gets confusing if we start mixing assertion libraries within a given plugin.